src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 40 * days;
+  let discount = 0;
```

Comment:
Use `const` instead `let`. You don't need to rewrite variables.
Just create variable for all values.  For discounts, amount of days, etc.